main/guides/zoe/README.md

```diff
@@ -159,6 +159,8 @@ t.is(typeof instance, 'object');
 
 _`makeIssuerKit` and `AmountMath.make` are covered in the [ERTP](../ertp/) section, along with `makeEmptyPurse`, `mintPayment`, and `getAmountOf` below._
 
+_For more information on the `E(zoe).startInstance` command please see the [Zoe API Documentation](https://docs.agoric.com/reference/zoe-api/zoe.html#e-zoe-startinstance-installation-issuerkeywordrecord-terms-privateargs)_
```

Comment:
```suggestion
_See also [E(zoe).startInstance(...)](/reference/zoe-api/zoe.md#e-zoe-startinstance-installation-issuerkeywordrecord-terms-privateargs)._
```

> your Markdown links should be to the .md Markdown files in the Doc repo. ...
> In general, use relative links instead of absolute ones for any links to files or folders in the Documentation repo.
-- [Writing Links](https://github.com/Agoric/documentation/blob/main/CONTRIBUTING.md#writing-links)

We should maybe go over [CONTRIBUTING](https://github.com/Agoric/documentation/blob/main/CONTRIBUTING.md) some time.